sklearn/metrics/cluster/_supervised.py

```diff
@@ -383,6 +390,11 @@ def adjusted_rand_score(labels_true, labels_pred):
 
     .. [wk] https://en.wikipedia.org/wiki/Rand_index#Adjusted_Rand_index
 
+    .. [Chacón22] J. E. Chacón and A. I. Rastrojo, Minimum adjusted Rand index
+      for two clusterings of a given size, 2022
+      https://arxiv.org/pdf/2002.03677.pdf
```

Comment:
I think we can link directly to Springer since it is open access there: https://link.springer.com/article/10.1007/s11634-022-00491-w

Reply:
I'm so used to assuming most sites are paywalled because I usually have institutional access to them saved via browser cache. Good catch, thanks.